src/pudl/dfc.py

```diff
@@ -79,10 +79,10 @@ def get(self, name: str) -> pd.DataFrame:
     def store(self, name: str, data: pd.DataFrame):
         """Adds named dataframe to collection and stores its contents on disk."""
         filename = self._get_filename(name, self._instance_id)
-        if not filename.startswith("gs://"):
+        if not filename.startswith("gs://") or not filename.startswith("s3://"):
```

Comment:
Either or is fine, just suggesting an alternative
```suggestion
        if not (filename.startswith("gs://") and filename.startswith("s3://")):
```

Reply:
oh this logic should probably be 
```suggestion
        if not (filename.startswith("gs://") or filename.startswith("s3://")):
```
i suspect this code path didn't get hit in my local `pudl_datastore`/`pudl_etl` experiments with this branch. 

One for if/when we add tests and try to integrate this upstream :)

Reply:
ah yes good point!